### PR TITLE
Use automat for article count

### PR DESCRIPTION
**Pull request by tomrf1**
Replaces frontend's implementation of article count with the shared version.

- Doesn't count repeat views of the same url
- Takes local storage as a parameter

See automat PR for details: https://github.com/guardian/automat-client-v2/pull/5

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/contributions-service.js [hunk 1]
```diff
@@ -6,12 +6,13 @@ import {
...
-import { fetchJson } from '../../../../lib/fetch-json';
-import { getCountryCode } from '../../../../lib/geolocation';
-import reportError from '../../../../lib/report-error';
+import { storage } from '@guardian/libs';
+import config from 'lib/config';
+import { getCookie } from 'lib/cookies';
+import fastdom from 'lib/fastdom-promise';
+import { fetchJson } from 'lib/fetch-json';
+import { getCountryCode } from 'lib/geolocation';
+import reportError from 'lib/report-error';
```
Comment (mxdvl) [inline static/src/javascripts/projects/common/modules/commercial/contributions-service.js:15]: Any reason for not using the relative paths?

https://github.com/guardian/frontend/pull/23494

File: static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import { storage } from '@guardian/libs';
-import config from '../../../../lib/config';
-import { elementInView } from '../../../../lib/element-inview';
+import { logView } from '@guardian/automat-contributions';
+import config from 'lib/config';
+import { elementInView } from 'lib/element-inview';
```
Comment (mxdvl) [inline static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js:4]: Same as above.

File: static/src/javascripts/projects/common/modules/commercial/contributions-service.js (hunk 1 above)
Comment (tomrf1) [reply static/src/javascripts/projects/common/modules/commercial/contributions-service.js:15]: oh right, I thought it looked nicer this way! I'll change

Comment (tomrf1) [reply static/src/javascripts/projects/common/modules/commercial/contributions-service.js:15]: Intellij is sad
Screen Shot 2021-08-13 at 09 03 22

Comment (mxdvl) [reply static/src/javascripts/projects/common/modules/commercial/contributions-service.js]: I agree, it’s generally nicer, we’ll only ask for files that end up in the commercial bundle.

**Review by tompretty — APPROVED** [review]
Looks good 👍

**Pull request merged**